### ISSUE TITLE
overloaded variable names

### ISSUE THREAD

**Issue by bakerfranke** [code-dot-org/dance-party#146]
I think this is a general thing but FWIW I was messing with this on level 11.

I created 5 dancers.  3 of them are named `new_dancer`.  All 5 appear, I can only control one.  Force user to make variable name on creation? auto-increment `new_dancer1`, `new_dancer2` etc.

![image](https://user-images.githubusercontent.com/11284819/48093927-57609b00-e1d6-11e8-8740-e2a227a91ec3.png)

**Comment by bakerfranke** [comment]
@ryansloan  yes.  it's not auto-incrementing when I drag from toolbox.  licecap evidence.
![out](https://user-images.githubusercontent.com/11284819/48210239-901a8480-e33c-11e8-83b8-59acb1688bfd.gif)

**Comment by boatmarina** [comment]
I think this is a Duplicate of #197
